src/components/CurrentCityForecast/CurrentCityForecast.module.scss

```diff
@@ -0,0 +1,74 @@
+@import '../../assets/styles/constants.scss';
+
+$current-temperature-font-size: 95px;
+$temperature-type-content: '℃';
```

Comment:
you also have something called degrees-content somewhere else? why?

Reply:
Done. I have moved this variable to constants.